app/_src/gateway/production/monitoring/ai-metrics.md

```diff
@@ -0,0 +1,104 @@
+---
+nav_title: Expose and Graphed Kong AI Metrics
+title: Expose and Graphed Kong AI Metrics
+minimum_version: 3.8.x
+---
+
+{:.note}
+> This feature requires {{site.ee_product_name}}.
+
+This guide walks you through getting and sending AI Metrics to Prometheus, and
+setting the Grafana Dashboard.
+
+## Overview
+
+Kong's AI Gateway (AI Proxy) call LLM-based services according to the settings of the AI Proxy,
+we will aggregate the LLM provider response to count the number of tokens used by the AI Proxy.
+If you have defined input and output cost in the models we will also calculate cost aggregation.
+The metrics details also exposed if the requests has been cached by Kong saving the cost of contacting
+the LLM providers thus saving performance and money.
+
+Expose metrics related to Kong and proxied upstream services in 
+[Prometheus](https://prometheus.io/docs/introduction/overview/) 
+exposition format, which can be scraped by a Prometheus Server.
+
+Metrics tracked by this plugin are available on both the Admin API and Status
+API at the `http://localhost:<port>/metrics`
+endpoint. Note that the URL to those APIs will be specific to your
+installation; see [Accessing the metrics](#accessing-the-metrics).
+
+This plugin records and exposes metrics at the node level. Your Prometheus
```

Comment:
```suggestion
The Kong [Prometheus plugin](/hub/kong-inc/prometheus/) records and exposes metrics at the node level. Your Prometheus
```
I had to guess at this based on other files, need to make it very clear on this page that the AI metrics require the Prometheus plugin.